### ISSUE TITLE
Console error related to video on homepage

### ISSUE THREAD

**Issue by alisonjoseph** [IBM-Design/design-website#278]
```
player.es.js:977 Uncaught (in promise) Error: #0062ff does not meet minimum contrast ratio. We recommend using brighter colors. (You could try #3281ff instead.) See WCAG 2.0 guidelines: http://www.w3.org/TR/WCAG/#visual-audio-contrast
    at VM170 7-e0cefe6d842905ca5b02.js:1
    at Array.forEach (<anonymous>)
    at k (VM170 7-e0cefe6d842905ca5b02.js:1)
    at s (VM170 7-e0cefe6d842905ca5b02.js:1)
(anonymous) @ player.es.js:977
k @ player.es.js:976
s @ player.es.js:1101
```

<img width="755" alt="screen shot 2019-01-31 at 2 15 37 pm" src="https://user-images.githubusercontent.com/2753488/52082551-b24e2800-2562-11e9-84cd-ab6c1dfbde29.png">

any ideas @jnm2377

**Comment by jnm2377** [comment]
@alisonjoseph So I set the Vimeo pause/play button to be the blue that the svg play icon was on hover, I think the token would be `brand-01` but I used the hex code for customizing that. We can totally remove it, or choose a different color. Without it, it just uses the default Vimeo colors.

It's `line 10` of `Video.js`
<img width="514" alt="screen shot 2019-01-31 at 4 34 02 pm" src="https://user-images.githubusercontent.com/32556167/52090131-18907600-2576-11e9-9090-42ade6d1025f.png">

**Comment by alisonjoseph** [comment]
oh weird. does the error go away if we remove? 🤔

**Comment by jnm2377** [comment]
I think so. It didn't have any errors when it was the Vimeo default colors.

**Comment by jnm2377** [comment]
I can remove it if we want it to just be default Vimeo colors.

**Comment by alisonjoseph** [comment]
yea lets just remove it
